spec/models/video_spec.rb

```diff
@@ -44,26 +41,43 @@
   
   describe :display_large_video_image do
     
+    let(:video) { object_generator(:video, small_cover_url: "/tmp/monk.jpg") }
+    subject { video.display_large_video_image }
+    
     it "should display a large video image if one is available" do
-      video = Video.new(title: "Monk", description: "Series about a detective", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
-      expect(video.display_large_video_image).to eq("/tmp/monk_large.jpg")
+      video.large_cover_url = "/tmp/monk_large.jpg"
+      expect(subject).to eq("/tmp/monk_large.jpg")
     end
   
     it "should display a placeholder image if there is no large video image"  do
-      video = Video.new(title: "Monk", description: "Series about a detective", small_cover_url: "/tmp/monk.jpg")
-      expect(video.display_large_video_image).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
+      expect(subject).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
     end
     
     it "should display a placeholder image if large_cover_url is an empty string" do
-      video = Video.new(title: "Monk", description: "Series about a detective", small_cover_url: "/tmp/monk.jpg", large_cover_url: "")
-      expect(video.display_large_video_image).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
+      video.large_cover_url = ""
+      expect(subject).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
     end
     
     it "should display a placeholder image if the large_cover_url path does not exist" do
-      video = Video.new(title: "Monk", description: "Series about a detective", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large2.jpg")
-      expect(video.display_large_video_image).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
+      video.large_cover_url = "/tmp/monk_large2.jpg"
+      expect(subject).to eq("http://dummyimage.com/665x375/000/fff.png&text=No+Preview+Available")
```

Comment:
Both of them will work because the url is hardcoded in my display_large_video_image method so it really just looking to match the string.  Turned off the wifi just in case and it worked